src/components/nameListStyle.css

```diff
@@ -0,0 +1,38 @@
+.nameList {
+    max-width: 280px;
+    /* border: 1px solid ; */
+    height: 280px;
+    margin: 10px;
+    background-color: azure;
+    text-align: left;
+    
+  }
+  .restaurant-image {
+    width: 200px;
+    height: 220px;
+  }
+  text-corner {
+    font-size: 13px;
+    position: relative;
+    padding: 3px;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    bottom: 185px;
+    left: 85%;
+    border: 1px solid;
+    border-radius: 40px;
+    background: rgb(224, 174, 129);
+  }
+  
+  .inlineStyle {
+    position: relative;
+    display: inline-flex;
+    text-align: left;
+
+  }
+  /* text {
+    font-size: 15px;
+    text-align: left;
+    width: 120px;
+  } */
```

Comment:
Leaving commented code is not good. It's always better to remove them 👍

Reply:
I Removed all comments, and change the name.